ereuse_devicehub/parser/computer.py

```diff
@@ -0,0 +1,446 @@
+import json
+import re
+from contextlib import suppress
+from datetime import datetime
+from fractions import Fraction
+from math import hypot
+from typing import Iterator, List, Optional, Type, TypeVar
+
+import dateutil.parser
+from ereuse_utils import getter as g
```

Comment:
Using `g` for `ereuse_utils.getter` could be missunderstood with `flask.g` (global)